Musakai.Shared/Api/ErrorResponses/E500/InternalServerErrorResponse.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Musakai.Shared.Api.ErrorResponses.E500;
```

Comment:
I think I will here use just ErrorResponses even if I keep it separated in the subfolders.... Same for E400